src/pages/About/AboutPage.style.js

```diff
@@ -0,0 +1,403 @@
+    width: 16.1875rem;
+    text-align: start;
+    white-space: pre-wrap;
+    ${reg18};
+    color: var(--grey100);
+    @media (max-width: 1279px) {
+      width: 20.9375rem;
+      margin-top: 2rem;
+      ${m_reg18}
+    }
+  `,
+
+  SubTitleContentContainer: styled.div`
+    display: flex;
+    flex-direction: column;
+    align-items: start;
+    gap: 0.5rem;
+  `,
+  SubTitle: styled.div`
+    ${bold36}
+    color: var(--grey100);
+    @media (max-width: 1279px) {
+      width: 20.625rem;
+      margin-top: 10rem;
+      ${m_bold36}
+    }
+  `,
+  SubTitleKr: styled.div`
+    ${reg18}
+    color: var(--grey300);
+    @media (max-width: 1279px) {
+      ${m_reg18}
+    }
+  `,
+  CardContainer: styled.div`
+    display: flex;
+    gap: 1.75rem;
+    @media (max-width: 1279px) {
+      display: flex;
+      flex-direction: column;
+    }
+  `,
+
+  HiddenCardContainer: styled.div`
+    display: flex;
+    gap: 1.75rem;
+    @media (max-width: 1279px) {
+      display: none;
+    }
+  `,
+
+  CardContentContainer: styled.div`
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    width: 10.1875rem;
+    height: 10.1875rem;
+    border-radius: 1rem;
+    background-color: var(--grey500);
+    box-sizing: border-box;
+    box-shadow: 0px 0px 20px 0px rgba(0, 0, 0, 0.4);
+    @media (max-width: 1279px) {
+      width: 20.9rem;
+      padding: 2.5rem 3.75rem;
+    }
+  `,
+  CardName: styled.div`
+    ${bold24}
+    color:var(--white);
+  `,
+  DepartmentContainer: styled.div`
+    display: flex;
+    flex-direction: column;
+  `,
+  DepartmentContentContainer: styled.div`
+    display: flex;
+    flex-direction: column;
+    max-width: 46.0625rem;
+    width: 40.0625rem;
+    margin-left: ${props => props.marginLeft || '0'};
+    padding: 2.5rem 3rem;
+    border-radius: 1rem;
+    background: #e3e6f0;
+  `,
+  DepartmentName: styled.div`
+    ${bold24};
+    color: var(--black);
+    margin-bottom: 0.52rem;
+    @media (max-width: 1279px) {
+      ${m_bold24};
+    }
+  `,
+  DepartmentMainDescription: styled.div`
+    margin-bottom: 1.5rem;
+    ${bold18};
+    color: var(--red);
+    @media (max-width: 1279px) {
+      ${m_bold18};
+    }
+  `,
+  DepartmentDescriptionContainer: styled.div`
+    display: flex;
+    justify-content: space-between;
+    @media (max-width: 1279px) {
+      flex-direction: column;
+      justify-content: start;
+      width: 100%;
+
+      gap: 1rem;
+    }
+  `,
+  DepartmentDetailsContainer: styled.div`
+    display: flex;
+    flex-direction: column;
+    gap: 0.5rem;
+    @media (max-width: 1279px) {
+      gap: 0rem;
+    }
+  `,
+  DepartmentDetailsTitle: styled.div`
+    ${bold18};
+    color: var(--black);
+    @media (max-width: 1279px) {
+      ${m_bold18};
+    }
+  `,
+  DepartmentDetails: styled.div`
+    white-space: pre-wrap;
+    ${reg14};
+    color: var(--black);
+    @media (max-width: 1279px) {
+      ${m_reg18};
```

Comment:
```suggestion
      font-size: 12px;
      font-weight: 300;
      line-height: 132%;
```
저번 리뷰에서 놓쳤는데 정해진 크기 없으면 피그마 그대로 써주면 됩니다~!!